File: dataforge-fx/src/main/java/hep/dataforge/fx/test/WebViewOutputTest.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package hep.dataforge.fx.test;

import hep.dataforge.fx.FXDataOutputPane;
import javafx.application.Application;
import static javafx.application.Application.launch;
import javafx.scene.Scene;
import javafx.scene.web.WebView;
import javafx.stage.Stage;

/**
 *
 * @author Alexander Nozik
 */
public class WebViewOutputTest extends Application {

    @Override
    public void start(Stage primaryStage) {

        WebView webView = new WebView();

        webView.getEngine().loadContent("a\tb\tc\naaaaa\tbbb\tccc");
        

        Scene scene = new Scene(webView, 400, 400);

        primaryStage.setTitle("WebViewOutputTest");
        primaryStage.setScene(scene);
        primaryStage.show();
    }

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        launch(args);
    }

}

File: dataforge-gui/src/main/kotlin/hep/dataforge/fx/test/WebViewOutputTest.kt
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package hep.dataforge.fx.test

import javafx.application.Application
import javafx.scene.Scene
import javafx.scene.web.WebView
import javafx.stage.Stage

/**
 *
 * @author Alexander Nozik
 */
class WebViewOutputTest : Application() {

    override fun start(stage: Stage) {

        val webView = WebView()

        webView.engine.loadContent("a\tb\tc\naaaaa\tbbb\tccc")


        val scene = Scene(webView, 400.0, 400.0)

        stage.title = "WebViewOutputTest"
        stage.scene = scene
        stage.show()
    }

}

fun main(args: Array<String>) {
    Application.launch(WebViewOutputTest::class.java, *args)
}